CHANGES.rst

```diff
@@ -124,6 +124,10 @@ Released 2021-05-11
 -   ``send_file`` sets the ``Content-Encoding`` header if an encoding is
     returned when guessing ``mimetype`` from ``download_name``.
     :pr:`3896`
+-   ``send_file`` sets the ``Content-Encoding`` header if an encoding is
```

Comment:
This needs to be in the 2.0.2 section. It can also be reduced to "`send_file` only sets a detected `Content-Encoding` if `as_attachment` is disabled."